src/test/regress/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[sql_base_schedule]
+insert_final_newline = false
```

Comment:
What's this one about?

Reply:
Our arbitrary config python scripts assume that all the non-empty lines contain a semicolon. Instead of fixing that python code, I took the liberty of adding an exception that saves the day.

https://github.com/citusdata/citus/blob/5b3537cdff35363461d30b745efaf60790cbe7a7/src/test/regress/citus_tests/arbitrary_configs/citus_arbitrary_configs.py#L124-L131

Reply:
Okay makes sense, can you add that as a comment to the .editorconfig file. Because it's not obvious why this is needed currently when only looking at the config.

Reply:
How about adding a simple check to allow empty lines in arbitrary config schedules and removing this exception that does not really make sense?

I did that in 8fe7d4cc5. This link may not work after a rebase

Reply:
I also updated the formatting of `sql_base_schedule` so that we have a final newline. You can see that change in a separate commit 9ea015217 that fixes this along with other new files with broken formatting after a merge from main.